Libraries/NativeAnimation/RCTNativeAnimatedNodesManager.m

```diff
@@ -74,6 +101,15 @@ - (BOOL)isNodeManagedByFabric:(nonnull NSNumber *)tag
   return false;
 }
 
+- (void)enqueueUpdateViewOnNativeThread:(nonnull NSNumber *)reactTag
```

Comment:
What about `enqueueUpdateViewOnUIManager`? As in use the ui manager to update this view props.

Reply:
Done!